pom.xml

```diff
@@ -454,6 +469,79 @@
         </plugins>
     </reporting>
     <profiles>
+        <!-- Enable source jars only if a corresponding safe license file is found - projects
```

Comment:
How do we define a safe license file? Is it by detecting presence of a license file?

Reply:
Yeah, the `<file>` check is maven's mechanism for enabling the profile if the specified file exists. This set is basically just a whitelist so we're pretty conservative about when we enable this.

Ideally we'd check the license in the pom itself, but it turns out maven has a pretty limited set of ways you can enable profiles built in, and extending that requires doing special things (putting extra jars on maven classpath) to make it work, so this seems a reasonable solution given the constraints.